fdbclient/ServerKnobs.cpp

```diff
@@ -869,7 +869,8 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( PERSIST_FINISH_AUDIT_COUNT,                             10 ); if ( isSimulated ) PERSIST_FINISH_AUDIT_COUNT = 1;
 	init( AUDIT_RETRY_COUNT_MAX,                                 100 ); if ( isSimulated ) AUDIT_RETRY_COUNT_MAX = 10;
 	init( SS_AUDIT_AUTO_PROCEED_COUNT_MAX,                         5 );
-	init( CONCURRENT_AUDIT_TASK_COUNT_MAX,                        50 ); if ( isSimulated ) CONCURRENT_AUDIT_TASK_COUNT_MAX = deterministicRandom()->randomInt(1, CONCURRENT_AUDIT_TASK_COUNT_MAX+1);
+	init( TESTING_AUDIT_STORAGE_THROTTLING,                    false );
```

Comment:
What is the benefit of adding this knob?

Reply:
We want to do throttling in real clusters but we do not want to do throttling when general simulation tests. In general simulation test, we will run auditStorage. We want to do the audit fast in this scenario.